app/services/email_alerts_updater.rb

```diff
@@ -11,6 +13,20 @@ def initialize(item:, successor:, govuk_request_id: nil)
     @govuk_request_id = govuk_request_id
   end
 
+  def handle
+    if successor.is_a?(ContentItem) && successor.document_collection_maps_to_tag?(item)
+      bulk_migrate
+    else
+      unsubscribe
+    end
+  end
+
+  def bulk_migrate
+    from_slug = subscriber_list_slug
```

Comment:
Good point. I've called it `item_subscriber_list` since it could also be a mainstream browse page (obviously not for the bulk migrate, but where we use it in the bulk_unsubscribe) and so this matches the language of this class